Fix existing CheckStyle issues (GlowChunkGenerator--GlowEntity)

Pr0methean opened the pull request.

Pr0methean committed:
Fix existing CheckStyle issues (GlowChunkGenerator--GlowEntity)

mastercoms commented on src/main/java/net/glowstone/chunk/GlowChunkSnapshot.java:
```diff
@@ -11,17 +12,31 @@
 public class GlowChunkSnapshot implements ChunkSnapshot {
 
-    private final int x, z;
+    @Getter private final int x;
```
Separate lines for annotations.

Pr0methean replied:
Done.

Pr0methean committed:
Move annotations to separate line

mastercoms commented on src/main/java/net/glowstone/net/GlowDatagramServer.java:
```diff
@@ -18,6 +18,12 @@
+     * Creates an instance for the specified server.
+     *
+     * @param server the associated GlowServer
+     * @param latch TODO: document this parameter
```
The countdown latch used during server startup to wait for network server binding.

mastercoms commented on src/main/java/net/glowstone/entity/GlowEntity.java:
```diff
@@ -150,11 +150,11 @@
     protected int ticksLived;
     /**
-     * Vehicle
+     * Vehicle.
```
The entity this entity is currently riding.

mastercoms commented on src/main/java/net/glowstone/entity/GlowEntity.java:
```diff
@@ -107,7 +107,7 @@
     protected final Vector velocity = new Vector();
     /**
-     * Passenger
+     * Passengers.
```
A list of entities currently riding this entity.

mastercoms commented on src/main/java/net/glowstone/entity/GlowEntity.java:
```diff
@@ -390,6 +391,10 @@ public void setVelocity(Vector velocity) {
     }
 
+    /**
+     * Returns the change in velocity per physics tick due to gravity.
```
Add a newline.

Pr0methean replied:
Done.

mastercoms commented on src/main/java/net/glowstone/entity/GlowEntity.java:
```diff
@@ -1447,6 +1506,16 @@ public Entity getLeashHolder() throws IllegalStateException {
     }
 
+    /**
+     * <p>Sets the leash on this entity to be held by the supplied entity.</p>
```
Don't use closing `p` tags, and add a new line between new paragraphs.

Pr0methean committed:
Implement @mastercoms' edits

Addresses:
https://github.com/GlowstoneMC/Glowstone/pull/646#discussion_r159073761,
https://github.com/GlowstoneMC/Glowstone/pull/646#discussion_r159073661,
https://github.com/GlowstoneMC/Glowstone/pull/646#discussion_r159073612,
https://github.com/GlowstoneMC/Glowstone/pull/646#discussion_r159073555,
https://github.com/GlowstoneMC/Glowstone/pull/646#discussion_r159073443

Pr0methean committed:
Merge remote-tracking branch 'refs/remotes/GlowstoneMC/dev' into style8
